src/BingAIClient.js

```diff
@@ -163,17 +163,16 @@ export default class BingAIClient {
             if (this.debug) {
                 console.debug(createNewConversationResponse);
             }
-            if (createNewConversationResponse.result?.value === 'UnauthorizedRequest') {
-                throw new Error(`UnauthorizedRequest: ${createNewConversationResponse.result.message}`);
-            }
             if (
                 !createNewConversationResponse.conversationSignature
                 || !createNewConversationResponse.conversationId
                 || !createNewConversationResponse.clientId
             ) {
                 const resultValue = createNewConversationResponse.result?.value;
                 if (resultValue) {
-                    throw new Error(`${resultValue}: ${createNewConversationResponse.result.message}`);
+                    const e = new Error(createNewConversationResponse.result.message); // default e.name is 'Error'
```

Comment:
@likev I think I'd prefer to keep the `resultValue` in the error message along with having it in `e.name`.

Reply:
Actually, I thought about it and never mind, your way is probably better.